src/audio/kpb.c

```diff
@@ -937,6 +939,24 @@ static bool kpb_has_enough_history_data(struct comp_data *kpb,
 	return buffered_data >= his_req;
 }
 
+static inline bool kpb_is_sample_width_supported(uint32_t sampling_width)
+{
+	bool ret;
+
+	switch (sampling_width) {
+	case 16:
+		ret = true;
+		break;
+	case 24:
+		ret = true;
+		break;
+	default:
+		ret = false;
+	}
```

Comment:
Although not needed, it's a common practice to add a break also on the default case.

Reply:
+1